### PR TITLE
Arcade crypto

### PR DESCRIPTION
**Pull request by f8daniel**
- altered advanced overlay to not close by clicking on background (really annoyed me because I have a tendency to accidentally click)
- made sure all games advanced options html include a button to close the advanced overlay
- improved dynamic display of advanced options in Blackjack
- added staking feature to Chess's advanced options
- fixed first draft code in game engine for general winner payment processing

TL;DR: users can attach a crypto stake to Chess games

### CODE REVIEW

**Comment by f8daniel** [comment]
Oh, and I guess I am pulling in all the changes to master that didn't pass through staging (even though they were synched as of 4 hours ago)
